R/mread.R

```diff
@@ -366,8 +368,7 @@ mread <- function(model, project = getOption("mrgsolve.project", getwd()),
     build[["nm-vars"]] <- nmv
     audit_nm_vars(
       spec,
-      param = param, 
-      init = init, 
+      x,
```

Comment:
I simplified this too ... no need to pass both when we can just pass the object `param` and `init` come from.